### DISCUSSION TITLE
FormItem DatePicker  编辑时 如何跟后端的字符串类型交互?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#45717] [Q&A]
formItem 上有个 normalize  可以支持 dayjs -> string 
  后端返回的 string -> dayjs 这步如何在组件 props 上处理呢?

**Comment by Wxh16144** [comment]
> 后端返回的 string -> dayjs 这步如何在组件 props 上处理呢?

处理好后再通过 initialValues 丢进去？

**Reply by vaynevayne** [reply]
有计划在 formItem 上 增加一个 prop 吗?

**Reply by vaynevayne** [reply]
formItem 加 parse 可能问题, 更好的是 datePicker 支持 string 类型的值 , 参考 element-ui

**Comment by crazyair** [comment]
参考 https://zhuanlan.zhihu.com/p/375753910 如何将 RangePicker 数组一分为二

**Reply by vaynevayne** [reply]
复杂度太高了, 不如自己处理数据

**Reply by crazyair** [reply]
这个是处理了2个事情，一个是 name 数组拆成了 start、end，一个是 moment 转时间戳，自己封装 Form.Item 一劳永逸的
如果只处理 moment 转时间戳，用 getValueProps + getValueFromEvent 即可
